inc/managers/class-event-manager.php

```
@@ -248,22 +253,24 @@
 
 		if ( ! $registered_event) {
-			return ['error' => 'Event not found'];
+			wu_log_add(self::LOG_FILE_NAME, 'Event not found', LogLevel::ERROR);
+			return false;
 		}
 
 		$payload_diff = array_diff_key(wu_maybe_lazy_load_payload($registered_event['payload']), $payload);
 
-		if (isset($payload_diff[0])) {
+		if (! empty($payload_diff[0])) {
 			foreach ($payload_diff[0] as $diff_key => $diff_value) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$diff_value'. (undefined)

(UnusedLocalVariable)